package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "metamask-crx",
-  "version": "10.28.1",
+  "version": "10.28.2",
   "private": true,
   "repository": {
     "type": "git",
```

Comment:
Changes to this file should not be included in this PR. Please revert the changes